library/src/main/java/com/northconcepts/templatemaster/rest/TemplateMasterBootstrap.java

```diff
@@ -164,6 +166,17 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         }
     }
 
+    @Override
+    public void requestInitialized(ServletRequestEvent event) {
+        HttpServletRequest request = (HttpServletRequest) event.getServletRequest();
```

Comment:
Let's make sure we're doing as much as we can here the we would normally do in the "before" part of `doFilter()` since it's not being called.

Reply:
Done